### ISSUE TITLE
LOOKING FOR ADNS2620 ??

### ISSUE THREAD

**Issue by biomurph** [biomurph/Mouse#7]
If you are looking for a Mouse Kit I have a small stash of ADS2620 in my lab. Please comment on this issue thread to learn more about how to get a kit or a sensor!

**Comment by axiomdata** [comment]
I would be interested in getting a full kit.

**Comment by calegxm999** [comment]
https://www.ebay.com.au/itm/1Pcs-ADNS-2620-A2620-DIP8-integrated-circuit/184068274587

**Comment by calegxm999** [comment]
Where can I get a kit?
Can I get one of ebay?

**Comment by biomurph** [comment]
You will need a bit more than just the ADNS2620 sensor. It also requires a plastic lens, and an LED with the correct frequency output. 

I don't have any complete kits, but I do have the LED, ADNS sensor, Lens, Resonator, and a few other parts necessary.
You would have to get your own PCB printed and assembled. It's not too hard a build, so even for a first timer, it's doable.

Realizing now that I didn't put up a BOM... I'll have to get to that shortly.

If you're seriously interested in getting the parts from me, reply to this thread and we can continue the conversation.

**Comment by briancolgan** [comment]
Seen as though the ADNS-2620 went EOL years ago, the project may need to migrate to a newer Pixart sensor?

Pixart have two mouse sensors in 8-pin DIP that can be used with red LEDs, PAW3220DB-TJDR and PAW3212DB-TJDT.

They look like like good alternatives to the ADNS-2620?

**Comment by biomurph** [comment]
@briancolgan 
Yep, Pixart is the place the picked up the Avago mouse sensor parts.
Are you a sales rep for Pixart??
I have been in touch with them, but they are more miss than hit when it comes to follow up with me. I think my project is pretty small.

And, thanks for the reminder, I will get back to them on the thread that I have going.

**Comment by briancolgan** [comment]
@biomurph Hi Joel, I work at EPS Global, we are Pixart's distributor in Europe.

For low volume projects below the Pixart MOQ, you can still buy their Sample Kits (Contents: 5 sets of Sensor + Lens + LED + Clip).

The **PAK3220RS-TJDR** is the sample kit for the PAW3220DB-TJDR sensor.
The **PAK3212RS-TJDT** is the sample kit for the PAW3212DB-TJDT sensor.

Hope that helps!

**Comment by biomurph** [comment]
@briancolgan 
Yes, that does help!
Can you connect me to Pixart's distribution in the US? I have been in contact with Open Source Hardware distributors and I want to sell this design as a kit. In order to get the to talk with me I need to have the BOM and supply chain worked out.

Thanks for reaching out and finding me! How did you get here, even?

**Comment by briancolgan** [comment]
@biomurph Pixart's distributor in US is WPG Americas http://www.wpgamericas.com/.

I found this project via EEVblog here: https://www.eevblog.com/forum/blog/eevblog-1244-mailbag/msg2677794/

I was reminded of it when I saw this thread on the ADNS-2620.
https://www.eevblog.com/forum/projects/source-for-an-optical-mouse-sensor-like-the-adns2620/msg3110516/

**Comment by biomurph** [comment]
Oh, Great! Glad to hear that the EEV Blog is getting folks over to me ;)
I just replied on a thread from last year, and a rep from WPG got back to me within 10 minues.
I'll see how it goes. Thanks again!

**Comment by justinclift** [comment]
Interesting.  Hadn't come across the "Mailbag" segment on EEVblog before.  Looks useful for aspiring projects trying to get noticed. :smile:

**Comment by justinclift** [comment]
> Glad to hear that the EEV Blog is getting folks over to me

They're covering the Tympan in the same episode too:

&nbsp; &nbsp; https://youtu.be/Uib8nUV1JEg?t=202

Ahhh, there was a bunch of stuff + included letter intro-ing it. :wink:
